Source/Core/HeadingPitchRoll.js

```diff
@@ -0,0 +1,96 @@
+/*global define*/
+define([
```

Comment:
For `define` here and in the tests, use the same style as other files, for example, see [Cartesian2](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/Cartesian2.js) for whitespace and list dependencies in alphabetical order.